src/components/MenuBase/MenuBase.tsx

```diff
@@ -3,15 +3,38 @@ import { createContext, ReactElement } from "react";
 import MUIMenu from "@mui/material/Menu";
 
 import { MenuBaseProps, MenuBaseContextProps } from "./MenuBaseTypes";
+import styled from "@emotion/styled";
+import theme from "@root/theme";
 
 export const MenuBaseContext = createContext<MenuBaseContextProps>(null);
 
+const StyledMUIMenu = styled(MUIMenu)`
+	& > div {
+		box-shadow: 0px 2px 6px rgba(26, 26, 26, 0.16);

```

Comment:
This is the same as our "theme.colors.almostBlack" with 16% opacity. Let's change it so if it ever changed in the future we don't have to go around manually updating it.

Reply:
If you click on the copy button it will automatically give you the property with the color and the opacity, all you'd need to do is update the color to our almostBlack.
![image](https://user-images.githubusercontent.com/23409284/193945985-11586c7f-e49f-40e5-9f2a-16a1126dd73c.png)